File: AgriculturaAuth.API/Models/ApplicationUser.cs
public class ApplicationUser
{
    public int Id { get; set; } // PK autoincremental
    public string Username { get; set; } = null!;
    public string Email { get; set; } = null!;
    public string FirstName { get; set; } = null!;
    public string LastName { get; set; } = null!;
    public string Telefono { get; set; } = null!;
    public string Role { get; set; } = null!;
}

File: product-service/src/main/java/com/agricultura/entity/ApplicationUser.java
package com.agricultura.entity;

import io.quarkus.hibernate.orm.panache.PanacheEntity;
import jakarta.persistence.*;

@Entity
@Table(name = "users")
public class ApplicationUser extends PanacheEntity {
    
    @Column(nullable = false, unique = true)
    public String username;
    
    @Column(nullable = false)
    public String email;
    
    @Column(name = "first_name", nullable = false)
    public String firstName;
    
    @Column(name = "last_name", nullable = false)
    public String lastName;
    
    public String telefono;
    
    @Column(nullable = false)
    public String role;
    
    public static ApplicationUser findByUsername(String username) {
        return find("username", username).firstResult();
    }
}